ml_engine/online_prediction/predict.cs

```diff
@@ -0,0 +1,117 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Google.Apis.Auth.OAuth2;
+using Newtonsoft.Json;
+
+namespace prediction_client
+{
+    class Person
+    {
+        public int age { get; set; }
+        public string workclass { get; set; }
+        public string education { get; set; }
+        public int education_num { get; set; }
+        public string marital_status { get; set; }
+        public string occupation { get; set; }
+        public string relationship { get; set; }
+        public string race { get; set; }
+        public string gender { get; set; }
+        public int capital_gain { get; set; }
+        public int capital_loss { get; set; }
+        public int hours_per_week { get; set; }
+        public string native_country { get; set; }
+    }
+
+    class MainClass
+    {
+        public static void Main(string[] args)
+        {
+            RunAsync().Wait();
+        }
+
+	static string project = "YOUR_PROJECT";
+	static string model = "census";  // Name of deployed model
+		
+        static HttpClient client = new HttpClient();
+
+        static async Task RunAsync()
+        {
+            client.BaseAddress = new Uri("https://ml.googleapis.com/v1/");
+            client.DefaultRequestHeaders.Accept.Clear();
+            client.DefaultRequestHeaders.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+
+            try
+            {
+                Person person = new Person
```

Comment:
Minor comment -- I'd expect you should be able to use `dynamic` here rather than have to define a Person class and use strong typing, just for filling in a bunch of fields.